django_lightweight_queue/runner.py

```diff
@@ -75,7 +85,7 @@ def handle_term(signum, stack):
                 else:
                     logger.info(
                         "Starting missing worker {} (exit code was: {})".format(
-                            worker.name,
+                            worker.name,  # type: ignore[attr-defined]
```

Comment:
> Is this because we set the name below?

Mostly; I think even if that were above this code it may still error because `name` isn't an attribute on a `Popen` object.

> Can this be a property on worker instead maybe?

Not sure what you mean by this?

> Or maybe if this is only used locally, we could have a mapping of names to workers that we use to get these?

Yeah, or we could recompute it each time from the `queue` and `worker_num`. Neither feels great though as there's a few places we use this.

Reply:
I mean maybe we could add an attribute/computed property/method to worker that has the name, rather than mangling it on.

Reply:
`worker` here is a `subprocess.Popen`, rather than a type which we control. I think that's why it's mangled on. I guess we could store `(worker, name)` where we currently just store `worker`?

Reply:
Ah apologies, I thought worker was a type we controlled. We could subclass Popen, or yeah just storing that might be good.

Reply:
c15d40d stores the name upfront.